keras_retinanet/bin/train.py

```diff
@@ -358,6 +364,13 @@ def main(args=None):
     # print model summary
     print(model.summary())
 
+    # this lets the generator compute backbone layer shapes using the actual backbone model
+    if "vgg" in args.backbone:
+        shapes_callback = make_shapes_callback(model)
+        train_generator.shapes_callback = shapes_callback
```

Comment:
@de-vri-es what do you think, should we wrap this in a function `set_shapes_callback()` or leave it like this? I'm a bit hesitant to modify attributes directly.

Reply:
I would stick with setting attributes directly. Much simpler and clearer to read.

We can even upgrade the thing to a property later if we really need to execute code when it changes.